sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeHealthcareEntitiesOptions.java

```diff
@@ -38,6 +40,18 @@ public AnalyzeHealthcareEntitiesOptions setIncludeStatistics(boolean includeStat
         return this;
     }
 
+    /**
+     * Gets the value of {@code disableServiceLogs}. The default value of this property is 'true'. This means,
+     * Text Analytics service won't log your input text. Setting this property to 'false', enables logging your input
+     * text for 48 hours, solely to allow for troubleshooting issues.
+     *
+     * @return true if logging input text service are disabled.
+     */
+    @Override
+    public boolean isServiceLogsDisabled() {
+        return disableServiceLogs == null ? true : disableServiceLogs;
```

Comment:
does Java usually sets the default in the client library?
In .NET, if the user didn't set a value for a parameter, we just don't pass it to the service, and let the service decide what the default is

Reply:
and why do you set it for 2 tasks and not the others?

Reply:
If it is an object `Boolean`, it is nullable. A primitive boolean can't be null.

The default value for a `Boolean` is `null`, the default value for a primitive `boolean` is `false`.

So since we already GAed with `boolean`, we can't change it to `Boolean`.

And the default value of `disableServiceLogs` is `true` in healthcare and pii tasks. So only these two need to set the default value to true

Reply:
Usually, we don't set the default value for an optional REST request parameter. If the type is `Boolean`, then we are the same behavior as .NET but unfortunately, a primitive `boolean` is GAed.

Reply:
I see. Do you know why Java went with `boolean`?

Reply:
I believed it is the uncaught attention that I missed. It is expected to have default to false at some point, but it is not the case anymore.

Reply:
No worries. I missed it because I didn't know about nullable in Java. But now we all know better and fixes are being made.
Thanks Shawn!